docs/source/guide/recommender_models.rst

```diff
@@ -0,0 +1,80 @@
+**Ranking models** are probably the most common use-case in recommender systems. When the community talks about recommender models, they probably mean ranking models. 
+
+The ranking model predicts a specific target. For example, it could be the review a customer would give an item or it could be the likelihood a user will click on an item. Click-through-rate prediction is a typical use-case for that. The Figure 1a above provides a simple example of a dataset. The dataset has different input features (``user_id``, ``user_age``, etc.) and the target (``click``). The dataset contains both the positive (``click=1``) and negative (``click=0``) interaction data. If negative examples are not available, then negative sampling strategies are required. An example architecture is visualized by Figure 1b. The inputs are concatenated and fed through an Multi-Layer Perceptron (MLP) Block (multiple fully connected layers).
+
+Merlin Models provides implementation of popular architectures, such as MLP, NCF, Facebook’s DLRM, Google’s DCN and Google’s Wide&Deep networks with TensorFlow backend. Classical machine learning models, such as implicit or XGBoost can be used in Merlin Models, as well.
+
+**Resources**:
+
+* `Exploring Different Models <https://github.com/NVIDIA-Merlin/models/blob/main/examples/03-Exploring-different-models.ipynb>`_ demonstrates how to build and train the popular deep learning architectures with **Merlin Models in TensorFlow** on an example dataset.
+* `Getting Started Guide - Ranking Models <https://github.com/NVIDIA-Merlin/Merlin/tree/main/examples/quick_start>`_ provides a detailed view on applying the ranking models to a dataset and how to use it for a new dataset.
+* `Serving Ranking Models <https://github.com/NVIDIA-Merlin/systems/blob/main/examples/Serving-Ranking-Models-With-Merlin-Systems.ipynb>`_ is a simple example how to deploy a ranking model with Merlin Systems and Triton in TensorFlow.
+* `Getting Started Guide - Ranking Models <https://github.com/NVIDIA-Merlin/Merlin/tree/main/examples/quick_start>`_ provides a detailed view on applying the ranking models to a dataset and how to use it for a new dataset.
+* `Getting Started Movielens <https://github.com/NVIDIA-Merlin/Merlin/tree/main/examples/quick_start>`_ is one end-to-end example for feature engineering, training and deployment with different frameworks (TensorFlow, PyTorch and HugeCTR) on a small scale.
+* `Scaling Criteo <https://github.com/NVIDIA-Merlin/Merlin/tree/main/examples/scaling-criteo>`_ is similar to the MovieLens example, but on a large scale of ~4 billion rows. It provides some guidance on multi-GPU scaling.
+
+Retrieval Models
+-------------
+
+.. image:: ./imgs/models_retrieval.png
+
+The goal of **retrieval models** are to retrieve hundreds of candidates from large item catalogs. In order to be scalable, they have a special neural network architecture: It usually processes the input data into two fully separated towers as visualized above Figure 2b. 
```

Comment:
I added the click columns back into it, because @gabrielspmoreira pointed out that you can train retrieval models on datasets with a target column and you can train ranking models which does not have a click column.

Negative sampling is an edge case, when we do not have a target and/or we have only positive interactions. Therefore, I kept the dataset the same and we will add a negative sampling section in the advanced method section. We have too much to write :)

Reply:
@bschifferer where does it say advanced methods section? can we at least add 1-2 liner and say that "one can train a retrieval model with and without a target column. When the explicit negatives are not available, i.e. we only have the positive user-item interaction data,  we can apply negative sampling techniques. Please visit advanced method section for more information on that".

if there is not advanced method may be you can write in the last sentence:

"we will talk more about negative sampling in our advanced method section (coming soon)."

Reply:
@mikemckiernan does not like to add sentences about sections which hasn't been written yet. (see his comment above).

I added sentences about negative sampling strategies:
`The dataset contains both the positive (click=1) and negative (click=0) interaction data. If negative examples are not available, then negative sampling strategies are required.`

`Similar than ranking models, retrieval models can be trained with negative sampling strategies when there are no negatives provided in the dataset.`